facia/conf/rome.properties

```diff
@@ -0,0 +1,2 @@
+rss_1.0.item.ModuleGenerator.classes=common.GModuleGenerator,common.RssAtomModuleGenerator
```

Comment:
Oddly it does not appear to be needed; the classpath is probably leaking across into dev build and preview.

Preview might need to be revisited in a real environment but we well probably be doing something visual rather than rendering RSS in real preview. So not a blocker/